component/app.jsonnet

```
@@ -3,7 +3,7 @@ local inv = kap.inventory();
 local params = inv.parameters.appcat_service_postgresql;
 local argocd = import 'lib/argocd.libjsonnet';
 
-local app = argocd.App('appcat-service-postgresql', params.namespace);
+local app = argocd.App('appcat-service-postgresql', params.namespace, secrets=false);
```

AFAIK `secrets` is already false by default (look into the function), or why setting it to false explicitly?
```suggestion
local app = argocd.App('appcat-service-postgresql', params.namespace);
```